### DISCUSSION TITLE
CTRL + <space>

### DISCUSSION THREAD

**Discussion by eBeyond** [wezterm/wezterm#6121] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'm using k9s and that has got a keybinding for CTRL+Space. But if I try to use it, it doesn't get it's way. In Karabiner-Elements which I need I can see in the EventViewer that it's clicked successfully. But when I start wezterm with the debug setting I can see all other events but not an event for CTRL+Space. 
Does that mean, that the event didn't reach wezterm or may it be the case that this event is silently ignored?

**Comment by bew** [comment]
(I see you marked your question as solved, did you found how to fix the issue?)

**Reply by eBeyond** [reply, marked as answer]
Yes...
I've tried it with iterm and it was the same. As such it's not a wezterm issue.
The solution is within the Systemsettings -> Keyboard -> shortcuts
There is a shortcut defined to switch the inputs. It's set to ^+<space> which means that this shortcut is caught by the OS and not passed to the terminal. After disabling the shortcut it works as expected.
